### ISSUE TITLE
Testing

### ISSUE THREAD

**Issue by paperscissors** [cashmusic/rackspace-template#29]
So @jeffsheltren @jessevondoom I tested Stripe on the Rackspace Template vagrant box in an embed, and I'm not getting that PHP-FPM error. It successfully runs through the transaction.

Notes on things I've found:

1. Subdomains don't point at user pages, they direct to the front public site (see screenshot)
2. SMTP info is loading from cashmusic.ini file into the platform, but emails aren't sending

![screenshot 2016-04-23 10 16 40](https://cloud.githubusercontent.com/assets/15576/14762965/9557c156-093c-11e6-9ff1-7fddcddf2549.png)

**Comment by jeffsheltren** [comment]
Regarding the SSL / PHP-FPM error, it's possible that's a bug introduced in a newer system package (be it curl, nss, or... something else).  Could you try the following:  login to the VM (```vagrant ssh```), and do a full system update: (```sudo yum -y update```).  Then reboot the VM and see if your Stripe test still works?  The test/stage servers have certainly been updated more recently than the Vagrant box image, so it's a bit behind on package versions.

For (1) above: does that problem exist even after you've added host entries for the subdomains you're testing?  e.g. you'll need to add ```10.10.10.20 dev.vagrant-multi1.cashmusic.org``` to work for the screenshot above.  I need to figure out with @jessevondoom the best way to deal with the subdomains (or disable them?) in the Vagrant environment.

For (2), what SMTP settings are you configuring in cashmusic.ini?

**Comment by paperscissors** [comment]
I just ran a yum update and I'm still not recreating that error. Here's a list of all the updates/installs:

  Updating   : kernel-firmware-2.6.32-573.22.1.el6.noarch                    
  Updating   : 1:facter-2.4.6-1.el6.x86_64      
  Updating   : tzdata-2016d-1.el6.noarch        
  Updating   : nss-softokn-freebl-3.14.3-23.el6_7.x86_64                     
  Updating   : bash-4.1.2-33.el6_7.1.x86_64     
  Updating   : glibc-2.12-1.166.el6_7.7.x86_64 
  Updating   : glibc-common-2.12-1.166.el6_7.7.x86_64                       
  Updating   : chkconfig-1.3.49.3-5.el6_7.2.x86_64                          
  Updating   : nspr-4.11.0-0.1.el6_7.x86_64    
  Updating   : nss-util-3.21.0-0.3.el6_7.x86_64
  Updating   : krb5-libs-1.10.3-42z1.el6_7.x86_64                          
  Updating   : libudev-147-2.63.el6_7.1.x86_64
  Updating   : nss-softokn-3.14.3-23.el6_7.x86_64                          
  Updating   : db4-4.7.25-20.el6_7.x86_64     
  Updating   : gawk-3.1.7-10.el6_7.3.x86_64   
  Updating   : grep-2.20-3.el6_7.1.x86_64     
  Updating   : coreutils-libs-8.4-37.el6_7.3.x86_64                        
  Updating   : coreutils-8.4-37.el6_7.3.x86_64
  Updating   : ca-certificates-2015.2.6-65.0.1.el6_7.noarch                
  Updating   : openssl-1.0.1e-42.el6_7.4.x86_64                            
  Updating   : nss-3.21.0-0.3.el6_7.x86_64    
  Updating   : nss-sysinit-3.21.0-0.3.el6_7.x86_64                         
  Updating   : nss-tools-3.21.0-0.3.el6_7.x86_64                           
  Updating   : openldap-2.4.40-7.el6_7.x86_64 
  Updating   : selinux-policy-3.7.19-279.el6_7.9.noarch                    
  Updating   : procps-3.2.8-35.el6_7.x86_64   
  Updating   : initscripts-9.03.49-1.el6.centos.4.x86_64                   
  Updating   : udev-147-2.63.el6_7.1.x86_64   
  Updating   : device-mapper-libs-1.02.95-3.el6_7.4.x86_64                 
  Updating   : device-mapper-1.02.95-3.el6_7.4.x86_64                      
  Updating   : device-mapper-event-libs-1.02.95-3.el6_7.4.x86_64           
  Updating   : openssh-5.3p1-114.el6_7.x86_64 
  Updating   : openssh-clients-5.3p1-114.el6_7.x86_64                      
  Updating   : perl-Git-1.7.1-4.el6_7.1.noarch
  Updating   : git-1.7.1-4.el6_7.1.x86_64     
  Updating   : device-mapper-event-1.02.95-3.el6_7.4.x86_64                
  Updating   : lvm2-libs-2.02.118-3.el6_7.4.x86_64                         
  Updating   : ntpdate-4.2.6p5-5.el6.centos.4.x86_64                       
  Updating   : 2:postfix-2.6.6-6.el6_7.1.x86_64                            
  Updating   : cronie-anacron-1.4.4-15.el6_7.1.x86_64                      
  Updating   : cronie-1.4.4-15.el6_7.1.x86_64 
  Updating   : kernel-headers-2.6.32-573.22.1.el6.x86_64                   
  Updating   : glibc-headers-2.12-1.166.el6_7.7.x86_64                     
  Updating   : glibc-devel-2.12-1.166.el6_7.7.x86_64                       
  Updating   : ntp-4.2.6p5-5.el6.centos.4.x86_64                           
  Updating   : lvm2-2.02.118-3.el6_7.4.x86_64 
  Updating   : openssh-server-5.3p1-114.el6_7.x86_64                       
  Updating   : puppet-3.8.6-1.el6.noarch      
  Installing : kernel-2.6.32-573.22.1.el6.x86_64                           
  Updating   : selinux-policy-targeted-3.7.19-279.el6_7.9.noarch           
  Updating   : libssh2-1.4.2-2.el6_7.1.x86_64 
  Updating   : 1:grub-0.97-94.el6_7.1.x86_64  
  Updating   : logrotate-3.7.8-26.el6_7.x86_64
  Updating   : kexec-tools-2.0.0-286.el6_7.2.x86_64                        
  Updating   : rpcbind-0.2.0-11.el6_7.x86_64  
  Updating   : db4-utils-4.7.25-20.el6_7.x86_64                            
  Updating   : htop-1.0.3-1.el6.x86_64        
  Updating   : libxml2-2.7.6-20.el6_7.1.x86_64
  Installing : kernel-devel-2.6.32-573.22.1.el6.x86_64   

So for 1), I did add it to the /etc/hosts file, but still gives me that issue. To simplify we might need to refer to the /user URIs instead of the subdomain on vagrant, if that's possible. Just to make it as simple as possible.

For 2), I'm dropping in my own SMTP creds--- I've tried on both SSL/TLS and non-SSL/TLS ports for good measure.
